src/Sulu/Bundle/SecurityBundle/Domain/Event/UserCreatedEvent.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\SecurityBundle\Domain\Event;
+
+use Sulu\Bundle\EventLogBundle\Domain\Event\DomainEvent;
+use Sulu\Bundle\SecurityBundle\Admin\SecurityAdmin;
+use Sulu\Component\Security\Authentication\UserInterface;
+
+class UserCreatedEvent extends DomainEvent
```

Comment:
would be nice to return the username from the `getResourceTitle` method in all user related events